src/recharge.iced

```diff
@@ -101,8 +101,10 @@ exports.Runner = class Runner extends Base
 
     fee = @fee_estimator { tx: tx }
 
-    if (@debug)
-      console.log('Expected total fee:', num*@min_amount() + fee)
+    if @debug()
```

Comment:
https://bitcoin.stackexchange.com/questions/1195/how-to-calculate-transaction-size-before-sending says for 1 input and 1 output the size will always be 148+34=182 +/- 10, which is what we expected